Dispatcher/UI/Screens/SplashPage/SplashViewController.swift

```diff
@@ -1,6 +1,9 @@
 import UIKit
 
 class SplashViewController: UIViewController {
+    
+    static var tabBarIdentifier = "TabBarController"
+    static var storyboardName = "Main"
```

Comment:
this is bad
we create identifiers only for cells 
this can be inside the scope that is being used
if this splash can lead to multiple places we will start having to maintain a list of all those identifiers